src/taggle.js

```diff
@@ -302,9 +308,11 @@
     Taggle.prototype._attachEvents = function() {
         var self = this;
 
-        _on(this.container, 'click', function() {
-            self.input.focus();
-        });
+        if (this.settings.focusInputOnContainerClick) {
+            _on(this.container, 'click', function() {
```

Comment:
Curious about this setting. When would you expect to click a "textarea" and not have it focus?